src/assignments/6-array.js

```diff
@@ -21,3 +30,14 @@ min=1
 
 */
 // start here writing the code for program2
+let j=0;
+let i=0;
+ const input = [1,4,2,3];
+ console.log(input.sort() );
+
+console.log(`max= ${input[input.length - 1]}`);
+console.log(`min= ${input[0]}`);
+
```

Comment:
good logic here ! :D 
but you can look at the other way how to do this: 
https://www.w3resource.com/c-programming-exercises/array/c-array-exercise-9.php